pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_publishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,36 @@
+include::../attributes.adoc[]
+[id="publishing-a-module-in-pantheon_{context}"]
+= Publishing a module in {ProductShortName}
+
+Publish a module in {ProductShortName} to the customer portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged-in.
+* The module to be published is uploaded.
+
+.Procedure
+
+. Navigate to link:https://pantheon.cee.redhat.com/#/search[].
+
+. Click the name of the module you want to publish.
++
+If you don't see the module listed, search for the modules using the search box.
+
+. In the list of module versions, click the three verical dots next to the version of the module you want to publish.
+
+. Click *Edit metadata* to add the following publishing details for the module:
++
+* *Product Name* - Select the Product and its Version that the module should be published for.
+* *Document use case* - Select the verb that best describes the module.
+* *Vanity URL fragment* - Enter a URL for the module.
+* *Search keywords* - Enter the words that, when searched by users, should bring up the module in the user's search. This is optional.
+
+* Click *Save*.
+
+. Click *Publish*.
++
+You can preview the module before publishing by clicking `Preview`.
```

Comment:
Yes, but we just instructed the user to click 'Publish', so it's too late for this notice. We might create a separate procedure for that, as simple as it might be for now, and include that task as part of a broader user story about preparing content for publication.

Reply:
Agree, and I see Previewing is a part of https://projects.engineering.redhat.com/browse/FCCEUD-28, so removing from here.